src/staging/src/utils/constants.js

```diff
@@ -9,6 +9,97 @@ export const PARK_NAME_TYPE = {
   Historic: 6,
 }
 
+export const countsList = [
+  // Use this to configure which counts show and in what order
+  // Don't show if isActive is false
+  {
+    display: "Reservable frontcountry sites",
+    countVar: "reservableSites",
+    isActive: true,
+  },
+  {
+    display: "Vehicle-accessible sites",
+    countVar: "vehicleSites",
+    isActive: true,
+  },
+  {
+    display: "Double sites",
+    countVar: "doubleSites",
+    isActive: true,
+  },
+  {
+    display: "Group sites",
```

Comment:
@ayumi-oxd you should change this to "Groupsites" to ensure that your other PR doesn't get lost when this is merged